storage_s3/indico_storage_s3/plugin.py

```diff
@@ -96,9 +96,15 @@ def getsize(self, file_id):
 
     def send_file(self, file_id, content_type, filename, inline=True):
         try:
+
+            content_disp = ('inline; filename="%s"' % filename if inline
```

Comment:
I would use werkzeug's Headers util to generate this string. While all our filenames should be sanitized to plain ascii without weird chars, it's better to have something that escapes such chars if necessary:

```
>>> from werkzeug.datastructures import Headers
>>> h = Headers(); h.add('Content-Disposition', 'inline', filename=u'x"y.png'); print h['Content-Disposition']
inline; filename="x\"y.png"
```